.gitignore

```diff
@@ -1,45 +1,46 @@
-{\rtf1\ansi\ansicpg1252\cocoartf1265\cocoasubrtf210
-{\fonttbl\f0\fnil\fcharset0 Consolas;}
-{\colortbl;\red255\green255\blue255;\red38\green38\blue38;}
-\margl1440\margr1440\vieww10800\viewh8400\viewkind0
-\deftab720
-\pard\pardeftab720
```

Comment:
I'm not sure why this stuff was here in the first place? Any idea?

Reply:
Yeah, so here's my best guess: I'm pretty sure this is an RTF header, which leads me to believe that this file was probably opened in TextEdit.app at some point. TextEdit can switch between RTF and plain text without making it super obvious, so it probably just got written back out as an RTF file after editing.